### DISCUSSION TITLE
Property 'getIdentitySchema' does not exist on type 'V0alpha2ApiInterface'

### DISCUSSION THREAD

**Discussion by AmirBrz** [ory/.github#17] [Q&A] [answered]
Hi Ory Guys
I want using Ory Kratos in a NestJs app (typescript project), I start writing test for warming up
But I have a problem with calling the [getIdentitySchema](https://github.com/ory/sdk/blob/master/clients/client/typescript/api.ts#L8909)
when i run the below test :
```
      it('Get Schema By Id', async () => {
        var result = await sdk.getIdentitySchema(schemaId.toString());
        console.log(result.data);
      })

```
I got this error :
_Property 'getIdentitySchema' does not exist on type 'V0alpha2ApiInterface'. Did you mean 'listIdentitySchemas'?_

consider the below test passed successfully, then seems the installed packages and configs is ok, and the both of them ([getIdentitySchema](https://github.com/ory/sdk/blob/master/clients/client/typescript/api.ts#L8909) & [listIdentitySchemas](https://github.com/ory/sdk/blob/master/clients/client/typescript/api.ts#L9115)) are members of [V0alpha2ApiInterface](https://github.com/ory/sdk/blob/master/clients/client/typescript/api.ts)

```
      it('Get Schema List By SDK', async () => {
        var result = await sdk.listIdentitySchemas();
        result.data.forEach(schema => {
          schemaId = schema.id ?? "";
          console.log(schema.id);
        })
      })
      //passed successfully
```
Where is the problem?
thank you

**Comment by vinckr** [comment, marked as answer]
Hello @AmirBrz 
Apologies for the late answer. 
We still need a good example for NestJS. 

Have you revisited Ory lately? We did have a stable release for the Ory Kratos SDK, so thinks might have improved in that area since. 

Ory Kratos SDK allows for integration with a self-hosted Ory Kratos Identity Server. Before using the SDK, I recommend to consult the Ory Kratos [REST API documentation](https://www.ory.sh/docs/kratos/reference/api). 

The source code for the generated SDKs can be found in the [Ory Kratos SDKs GitHub repository](https://github.com/ory/sdk/tree/master/clients/kratos/).
For managing identity schemas, Ory provides a way to customize and manage them according to your needs. You can define which field is used as the identifier when logging in, and fields which are used to verify or recover the user's identity.

For more specific information about the getIdentitySchema method and how to use it in your NestJs app, I recommend checking the [Ory SDK documentation](https://github.com/ory/sdk/blob/master/clients/client/typescript/api.ts) or the [Ory Kratos SDKs GitHub repository](https://github.com/ory/sdk/tree/master/clients/kratos/).
